airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/RouteToTaskQueueActivityImpl.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.sync;
+
+import io.airbyte.config.Configs;
+import io.airbyte.workers.temporal.TemporalJobType;
+import java.util.UUID;
+
+public class RouteToTaskQueueActivityImpl implements RouteToTaskQueueActivity {
+
+  // To be replaced by proper Routing Service. For now, our MVP supports a single external Data Plane
+  private static final String MVP_DATA_PLANE_TASK_QUEUE = "MVP_DATA_PLANE";
+
+  private final Configs configs;
+
+  public RouteToTaskQueueActivityImpl(final Configs configs) {
+    this.configs = configs;
+  }
+
+  @Override
+  public String routeToTaskQueue(final UUID connectionId) {
+    if (configs.connectionIdsForDataPlane().contains(connectionId.toString())) {
```

Comment:
can you add a comment to explain why this logic makes sense? as a reader,  i'm missing some piece of context.

Reply:
Continued in child PR: https://github.com/airbytehq/airbyte/pull/15905/files#r953199345